skimage/measure/profile.py

```diff
@@ -86,14 +90,31 @@ def profile_line(image, src, dst, linewidth=1,
            [1.        , 1.        , 0.        ],
            [1.41421356, 1.41421356, 0.        ]])
     """
+    image_is_bool = image.dtype == bool
+
+    if order is None:
+        if image_is_bool:
+            order = 0
+        else:
+            order = 1
+
+    if image_is_bool and order != 0:
+        warn("Input image dtype is bool. Interpolation is not defined "
+             "with bool data type. Please set order to 0 or explicitely "
+             "cast input image to an other data type.")
```

Comment:
stacklevel=2?

also: an other -> another

Finally, I wonder whether we should simply raise here, since the output is liable to be erroneous here...

Reply:
I am not against raising but it is a little bit aggressive. May be can we then start a deprecation cycle specifying that starting from version 0.19 an error will be raised in this case.?